FogCreek/Glitch-Community
# Superb guide: Componentize new stuff stuff

## Links
https://glitch.fogbugz.com/f/cases/3328475/Componentize-the-new-stuff-overlay
https://superb-guide.glitch.me/storybook/index.html?path=/story/new-stuff--article

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/8932174/57100270-80c53400-6cec-11e9-9d6b-a252caa277a8.png)

## Changes:
- Components for new stuff articles and the pup
- Simplified css for the pup in the corner when there's unread stuff
- Bumped up the css priority on the background for TransparentButton because thee css for presenter button:hover beat it out

## How To Test:
- Check out the stories, and look at the new stuff overlay
- Use `localStorage.removeItem('community-userPrefs')` to bring the dog back
